### ISSUE TITLE
Rename a node as a sub-directory, and make directory as new node type.

### ISSUE THREAD

**Issue by exaos** [Kungsgeten/org-brain#172]
I'd like to organize my nodes into hierarchical directories, e.g.:
> ─── vivo
>     ├── arts
>     ├── history
>     ├── languages
>     ├── physiology
>     ├── psychology
>     ├── society
>     └── special

I have an old entry named "arts" which is stored as file "arts.org". When I try "org-brain-rename-file" to rename it as "vivo/arts/index", I have to correct some links and names mannually. So, I hope that we can make some filenames special, or we can add subdirectory as a special entry type.
- When we look for `dir1/dir2/`, the file `dir1/dir2/index` will be automatically found as the node `dir1/dir2`. Links to `dir1/dir2/index` can be shortened as `dir1/dir2/`.
- Add a function to rename a file node into a directory node.

I really need this feature. When the number of nodes expands quickly, hierarchical directories would be very helpful.

Thanks for the great work of `org-brain`!

**Comment by exaos** [comment]
Another question: how about making file entries within one directory use relative links?

**Comment by Kungsgeten** [comment]
Sorry but I don't think I quite understand what you're asking for. You want to have a shorthand when linking to file entries, that if you end with a `/` then it will append `index`? `org-brain` should handle the links for you though, so I'm not sure I see the benefits (there should be no need for manual renaming, even though there might be needs for it now due to the software being buggy or incomplete).

What do you mean by relative links, in your second question?

**Comment by MarioRicalde** [comment]
This issue has been stale since July 22, 2019. Perhaps it's time to archive it.

**Comment by Kungsgeten** [comment]
@MarioRicalde @exaos Yes maybe. On the other hand (if I remember correctly) there has been other discussions about making subdirectories "special" in some way. Maybe these discussions should be merged into one issue, and discussions could continue there...
